### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by roodjinocherilus**
Desktop Version of the Portfolio Project

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -17,22 +17,37 @@
...
                 <div class="text">
-                    <h1>I’m Roodjino Glad to see you!</h1>
+                    <h1>I’m Roodjino, Glad to see you!</h1>
                     <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
                 </div>
-                <div class="connect">
+                <div class="connect connect2">
                     <p> LET'S CONNECT</p>
                     <ul class="logo"> 
                         <li><a href="#"><img src="images/twitter.svg"> </a> </li>
```
Comment (ElsonOtake) [inline index.html]: - [x] Please align this text to the left side as in the template provided by Figma. Your design must stick to the template as a project requirement.

Figma template
Screen Shot 2022-11-05 at 10 20 31 PM

Your design
Screen Shot 2022-11-05 at 10 19 55 PM

File: index.html [hunk 2]
```diff
@@ -74,11 +89,11 @@ <h2>Project Title</h2>
 
             
             <div class="card-work">
```
Comment (ElsonOtake) [inline index.html:91]: - [x] Kindly switch the place of the image for each project. One image to the left of the text and the following with the image to the right of the text, and so forth. Stick to the design supplied by Figma.

Figma template
Screen Shot 2022-11-05 at 10 23 17 PM

Your design
Screen Shot 2022-11-05 at 10 23 40 PM

File: index.html [hunk 3]
```diff
@@ -74,11 +89,11 @@ <h2>Project Title</h2>
...
-                <div class="card-img">
+                <div class="card-img card-img2">
                     <img src="images/portfolio2.png" alt="snapshoot">
                 </div>
                     
-                <div class="card-text">
+                <div class="card-text card-text2">
                     <h2>Project Title</h2>
                     <ul class="project-details">
                         <li>Canopy</li>
```
Comment (ElsonOtake) [inline index.html:95]: - [x] Kindly use the same gray circle on the line of "Back end dev" as you did on the first card. Sticking to the template is a project requirement.

Screen Shot 2022-11-05 at 10 28 38 PM

File: index.html [hunk 4]
```diff
@@ -149,9 +164,9 @@ <h2>Project Title</h2>
 
         <section class="about">
 
-            <div class="container">
-                <div class="text">
-                    <h1>I’m Roodjino Glad to see you!</h1>
+            <div class="container container2">
+                <div class="text text-about">
+                    <h1>About Myself</h1>
```
Comment (ElsonOtake) [inline index.html:175]: - [ ] Please align the "Let's connect", the icons, and the button with the text above it. Sticking to the template is a project requirement.
- [x] Please create horizontal lines below "Frameworks" and "Skills" as in the template. Sticking to the template is a project requirement.
- [x] Kindly stretch the area used by languages as in the template. Sticking to the template is a project requirement.

Figma template
Screen Shot 2022-11-05 at 10 39 54 PM

Your design
Screen Shot 2022-11-05 at 10 33 38 PM

File: index.html [hunk 5]
```diff
@@ -187,7 +202,9 @@ <h1>I’m Roodjino Glad to see you!</h1>
 
         </section>
 
+        <section class="form">
```
Comment (ElsonOtake) [inline index.html:211]: - [x] Kindly remove the border from the input fields following the design supplied. Sticking to the design is a project requirement.
- [x] Please style your button with a similar padding space as in the design provided. Sticking to the design is a project requirement.

Figma template
Screen Shot 2022-11-05 at 10 42 59 PM

Your design
Screen Shot 2022-11-05 at 10 43 22 PM

File: index.html [hunk 6]
```diff
@@ -17,22 +17,37 @@
...
                     </li>
-
+                
                 </ul>
             </nav>
 
         </header>
 
-        <section>
+        <section class="hero">
```
Comment (ElsonOtake) [inline index.html:44]: - [x] I think it's better to use the whole width of the page to present your content. Your page is presenting a blank right side when in the desktop version, creating a nonfriendly UX.

Screen Shot 2022-11-05 at 10 49 09 PM

**Review by ElsonOtake — CHANGES_REQUESTED** [review]
STATUS: CHANGES REQUIRED 🟡 

Hi @roodjinocherilus @emmanuelcortes2 ,

Good job so far! 🎉 🎉 
There are some issues that you still need to work on to go to the next project but you are almost there! 🥲 🥲 

woohoo-youre-almost

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️
- Professional README file. ✔️
- PR has a good title and summary. ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by roodjinocherilus** [comment]
> STATUS: CHANGES REQUIRED 🟡
> 
> Hi @roodjinocherilus @emmanuelcortes2 ,
> 
> Good job so far! 🎉 🎉 There are some issues that you still need to work on to go to the next project but you are almost there! 🥲 🥲
> 
> woohoo-youre-almost
> 
> ## Highlights
> * No linter errors. ✔️
> * GitHub flow is followed. ✔️
> * Professional README file. ✔️
> * PR has a good title and summary. ✔️
> 
> ## Required Changes ♻️
> _Check the comments under the review._
> 
> ## Optional suggestions
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Thank you for your review

File: index.html [hunk 7]
```diff
@@ -198,10 +221,10 @@ <h2>Contact me</h2>
                 <input type="email" id="email" placeholder="Enter your Email"  minlength="2" required >
                 <textarea name="textarea" id="message" placeholder="Write your Message here" required  minlength="2" maxlength="500"></textarea>
                     <button class="btn-contact" type="submit" required>Get in touch</button>
```
Comment (topeogunleye) [inline index.html:223]: - [ ] [OPTIONAL] On hover state, I kindly suggest that you add a border-color of white around your button to make sure your button color does not merge with your contact form background-color

**Review by topeogunleye — APPROVED** [review]
### Hi @roodjinocherilus, 

# Status: Approved ✔️
Excellent job on making the changes requested by the previous code reviewer. Keep it up 👏

Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉
congratulations
>
## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Followed Github flow :heavy_check_mark:
- Professional README file. :heavy_check_mark:
 
## [Optional] suggestions:
* [ ]  Please check my inline comments.
 
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**